ButterflyMX Demo/Logic/CallNotificationTypeManager.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  CallNotificationTypeManager.swift
+//  ButterflyMX Demo
+//
+//  Created by Yingtao Guo on 11/18/21.
+//  Copyright © 2021 Taras Markevych. All rights reserved.
+//
+
+import Foundation
+
+enum CallNotificationType: Int {
+    case videoCall
```

Comment:
both of type accept video, it's more related to push delivery type, like `callKit , pushNotifications`

Reply:
fixed it.